.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
can you remove the file, use insted local config

Reply:
What do you mean by local config? This file is being used by your editor only if you have prettier plugin installed.

Reply:
instead use vscode user config, not for repo

Reply:
Your choice. I will remove it.